src/DynamicMvvm.Abstractions/ViewModel/IViewModel.Extensions.Properties.cs

```diff
@@ -248,7 +248,7 @@ public static IDynamicProperty GetOrResolveProperty(this IViewModel viewModel, s
 
 			if (!viewModel.TryGetDisposable<IDynamicProperty>(name, out var property))
 			{
-				typeof(IViewModel).Log().LogWarning($"Resolving property '{viewModel.GetType().Name}.{name}' using reflection on '{viewModel.Name}'.");
+				typeof(IViewModel).Log().LogViewModelResolvingPropertyUsingReflection(viewModel.GetType().Name, name, viewModel.Name);
```

Comment:
something I'm wondering is when we use the method `Log()`, does it always recreate an instance of logger? If so, couldn't it be also decrease the perf?

Reply:
It's probably a singleton or something static?

Reply:
I asked myself the same question! It's another thing to investigate, but it's out of scope for this PR.